etc/answers-headless.api.md

```diff
@@ -498,6 +498,11 @@ export enum SearchTypeEnum {
     Vertical = "vertical"
 }
 
+// @public @deprecated
+export interface SelectableFilter extends Filter {
+    selected: boolean;

```

Comment:
After discussing this approach with Tom, we came to the conclusion that it would be simpler to add an optional `displayName` to `SelectableFilter` rather than deprecating `SelectableFilter` and creating `DisplayableFilter`. Strictly speaking, adding an optional property is breaking change, however it is less much less likely to cause issues compared to removing `SelectableFilter` altogether.

Reply:
I see. This approach wouldn't have consistent naming like what we have with `DisplayableFacet` from core. If we are introducing some type of breaking change either way, I think we should rename altogether in the next major version. But yea this would make it complicated for backward compatibility so adjusting SelectableFilter to have displayName make sense. I will update :+1: